utils/get_supported_scylla_base_versions.py

```diff
@@ -95,8 +95,18 @@ def get_supported_scylla_base_versions(self, supported_versions) -> list:  # pyl
                     # The dest version is a minor release of latest enterprise release
                     ent_base_version.append(version)
                 else:
-                    # Choose the last two releases as upgrade base
-                    ent_base_version += ent_release_list[idx-1:][:2]
+                    lts_version = re.compile(r'\d{4}\.1')  # lts = long term support
+                    sts_version = re.compile(r'\d{4}\.2')  # sts = short term support
```

Comment:
And what if we have 2023.3 ?

Is that the definition of what LTS shouldn't be coming from SCT code like that...

Reply:
first, i don't know if we plan to have such, but if we do, then it will need to find last version + last stable version... and also the other ones will need to be changed... so for example if got `2023.3`, and now when working on `2024.1` we will need to support last version and last LTS...
for now, AFAIU we won't have, and if we do, we have to understand what it will be, and what is the support we are supposed to have